packages/integrations/vercel/src/lib/web-analytics.ts

```diff
@@ -0,0 +1,56 @@
+import type { AnalyticsProps } from '@vercel/analytics';
+import { fileURLToPath } from 'url';
+
+export type VercelWebAnalyticsConfig = {
+	enabled: boolean;
+	config?: Omit<AnalyticsProps, 'beforeSend'>;
+};
+
+async function getWebAnalyticsFunctions(root: URL) {
+	try {
+		const files = await Promise.all([
+			import(/* @vite-ignore */ fileURLToPath(new URL('./vercel-web-analytics.ts', root))).catch(
+				() => undefined
+			),
+			import(/* @vite-ignore */ fileURLToPath(new URL('./vercel-web-analytics.js', root))).catch(
+				() => undefined
+			),
+		]);
+
+		const functions = files[0] || files[1];
+
+		if (functions?.default) {
+			if (typeof functions.default.beforeSend !== 'function') {
```

Comment:
The example that you wrote doesn't export a `default`. I think it's best **to not rely on `default`**. Without a `default`, we can make our APIs more flexible if, in the future, we want to add more exports.

Reply:
Thanks for the heads up. I was not using `default` at all, I think it was a left over from testing. Totally agree, hence why it's not using `default`